src/infrastructure/config.py

```
@@ -0,0 +1,146 @@
+            values['debug'] = True
+            if 'logging' in values:
+                values['logging'].level = 'DEBUG'
+        return values
+
+    @classmethod
+    def from_file(cls, config_path: Path) -> 'ApplicationConfig':
+        import json
+        import yaml
+        
+        if not config_path.exists():
+            raise FileNotFoundError(f"Config file not found: {config_path}")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)